frontend/src/features/queue/helpers/isStaffUser.ts

```diff
@@ -0,0 +1,14 @@
+import {
+  IDIR_USER_ROLE,
+  IDIRUserRoleType,
+} from "../../../common/authentication/types";
+import { Optional } from "../../../common/types/common";
+
+const staffRoles: IDIRUserRoleType[] = [
+  IDIR_USER_ROLE.CTPO,
+  IDIR_USER_ROLE.PPC_CLERK,
+  IDIR_USER_ROLE.SYSTEM_ADMINISTRATOR,
+];
+
+export const isStaffUser = (userRole: Optional<IDIRUserRoleType>) =>
```

Comment:
As mentioned at the beginning of this review, I think this method should be renamed as "isStaffUser" is misleading (since other staff users without access to the queue page are still staff users)